### DISCUSSION TITLE
Attempting to access settings at main.py but can't

### DISCUSSION THREAD

**Discussion by jferris-** [itsToggle/plex_debrid#416] [Q&A]
I recently toggled this;
> If you dont want the main menu to show when starting the script (for an auto-run setup), navigate to "/Settings/UI Settings/show menu on startup" and set the value to "false". 

My issue now that I'd like to access the menu again, I can't.

Typing _python main.py_ into the terminal just kick starts plex_debrid, skipping the menu.
Typing 'exit' displays 'Waiting for the download automation to stop ...' but it never stops...

How can I resolve this, please?

**Comment by itsToggle** [comment]
you cant kill threads within python, so you will just have to wait a minute or two until the inital scan is done. After that it will return to the main menu.
You could also edit the settings.json file directly.
